changelogs/unreleased/gh-8269-log-syslog-incorrect-facility.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/log
```

Comment:
Nits regarding the commit message:

* Subject formatting and wording:
  * `log: Fixed` -> log: fix` (title case -> lower case, past tense -> imperative mood).
  * `in syslog in docker and other cases` -> `in syslog on Alpine and OpenBSD`
  * But the subject line is anyway too long, so I would rephrase it:

    > log: fix syslog facility for musl and OpenBSD
* Typo: `Alipine` -> `Alpine`.
* `bug with syslog severity` -> `bug with syslog priority`.
* `Follow-up #8244` -- it should be removed, GitHub links the PR from the commit on its own.

Reply:
Fixed.